Ubuntu weekly build failed:

```
CMake Error at cmake_modules/BuildUtils.cmake:297 (target_link_libraries):
  Target "arrow_testing_objlib" links to:

    Boost::process

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  src/arrow/CMakeLists.txt:1124 (add_arrow_lib)
```

@zhouyuan, do you happen to know the reason? I remember the main branch has a similar issue before your fix.

--- a/ep/build-velox/src/get-velox.sh
+++ b/ep/build-velox/src/get-velox.sh
@@ -65,7 +65,7 @@ if [ "$VELOX_HOME" == "" ]; then
 fi
 
 function process_setup_ubuntu {
-  sed -i "s|run_and_time install_arrow||g" scripts/setup-centos9.sh
+  sed -i "s|run_and_time install_arrow||g" scripts/setup-ubuntu.sh
   echo "Using setup script from Velox"
 }
 

